.github/workflows/commit-workflow.yml

```diff
@@ -130,7 +130,7 @@ jobs:
         # https://github.com/marketplace/actions/rust-cargo
         with:
           command: tarpaulin
-          args: -v --out Xml
+          args: --avoid-cfg-tarpaulin -v --out Xml
```

Comment:
Can you say why this is necessary?

Reply:
Yes, tarpaulin gives an error to bitvec 0.19.5 if the toolchain version is 1.52 or higher. People reported this in https://github.com/xd009642/tarpaulin/issues/756, and the solution is to add the argument.